src/index.ts

```diff
@@ -220,7 +225,10 @@ export async function createOrBuild(
   enableV3?: boolean | undefined,
   dbEncryptionKey?: Uint8Array | undefined,
   dbDirectory?: string | undefined,
-  historySyncUrl?: string | undefined
+  historySyncUrl?: string | undefined,
+  walletType?: WalletType | undefined,
+  chainId?: number | undefined,
+  blockNumber?: number | undefined
```

Comment:
Don't think we need the block number here either. This only matters when you are actually signing something.